openlibrary/plugins/openlibrary/api.py

```diff
@@ -487,3 +488,36 @@ def response(msg, status="success"):
             }), content_type="application/json")
 
         return response('Observations removed')
+
+
+class work_delete(delegate.page):
+    path = r"/works/(OL\d+W)/delete"
```

Comment:
Because our URLs have human readable slugs (e.g `/works/OL1W/Harry_Potter`), we can't have URLs like `/works/OL1W/delete`, cause it would cause issues with books called Delete :P . We'll need to ignore the human readable slug:

```suggestion
    path = r"/works/(OL\d+W)/[^/]+/delete"
```

And when we call this API, we call it with `/works/OL1W/-/delete` (because the human readable slug can actually be any string).